tests/complete/main.tf

```diff
@@ -63,14 +63,41 @@ module "eks" {
   version = "~> 19.13"
 
   cluster_name                   = local.name
-  cluster_version                = "1.25"
+  cluster_version                = "1.26"
   cluster_endpoint_public_access = true
 
   vpc_id     = module.vpc.vpc_id
   subnet_ids = module.vpc.private_subnets
 
   manage_aws_auth_configmap = true
 
+  cluster_addons = {
+    aws-ebs-csi-driver = {
+      most_recent              = true
+      service_account_role_arn = module.ebs_csi_driver_irsa.iam_role_arn
+    }
+    coredns = {
+      preserve    = true
```

Comment:
I believe this will fail on a new `apply` - https://github.com/hashicorp/terraform-provider-aws/issues/27481 - in the v5 AWS provider there will be two different fields, but in the current provider this will fail (the AWS API doesn't support `preserve` when making the create API call)